include/video/driver.hpp

```diff
@@ -43,10 +43,11 @@ class VideoDriver {
     VideoDriver& operator=(const VideoDriver&) = delete;
 
     virtual ~VideoDriver();
-    virtual Point     get_mouse()         = 0;
-    virtual InputMode input_mode() const  = 0;
-    virtual int       scale() const       = 0;
-    virtual Size      screen_size() const = 0;
+    virtual Point     get_mouse()                   = 0;
+    virtual InputMode input_mode() const            = 0;
+    virtual int       scale() const                 = 0;
+    virtual Size      screen_size() const           = 0;
+    virtual void      set_fullscreen(const bool on) = 0;
```

Comment:
`const` isn’t meaningful in this context (function declaration argument).

I believe you can keep it in the function definition if you really want, but please remove it from the declarations.